ffxivbot/handlers/QQGroupCommand_hunt.py

```diff
@@ -0,0 +1,455 @@
+                    msg = "狩猎时钟list命令示例：\n/hunt list [选项]\n选项解释：\ncd：列出可触发的s"
+                          # "和其开始时间、必定触发时间、进度\n50s：列出所有2.0的s怪和其开始时间\n60s：列出所有3.0的s怪和其开始时间\n70s：列出所有4.0的s怪和其开始时间"
+
+            elif (optype == "maintained"):
+                try:
+                    a = 1
+                    if group_info.servermark == "myc":
+                        while a < 30:
+                            ktinfo = ServersMonsterKillTime.objects.get(monsterid=a)
+                            ktinfo.myc = time.time()
+                            ktinfo.mycm = 1
+                            ktinfo.save()
+                            a += 1
+                    elif group_info.servermark == "syzd":
+                        while a < 30:
+                            ktinfo = ServersMonsterKillTime.objects.get(monsterid=a)
+                            ktinfo.syzd = time.time()
+                            ktinfo.syzdm = 1
+                            ktinfo.save()
+                            a += 1
+                    msg = "击杀时间已设置为当前时间，并修改触发时长为维护后触发时长"
+                except ServersMonsterKillTime.DoesNotExist:
+                    msg = ""
+
+            elif (optype == "edit"):
+                try:
+                    monster = param_segs[1].strip()
+                    YMD = param_segs[2].strip()
+                    HMS = param_segs[3].strip()
+                    try:
+                        monster_info = MonsterList.objects.get(monstername=monster)
+                        try:
+                            ktinfo = ServersMonsterKillTime.objects.get(monsterid=monster_info.monsterid)
+                            edittimestr = YMD + " " + HMS
+                            edittime = int(time.mktime(time.strptime(edittimestr, TIMEFORMAT_YMDHMS)))
+                            if group_info.servermark == "myc":
+                                ktinfo.myc = edittime
+                                ktinfo.save()
+                            elif group_info.servermark == "syzd":
+                                ktinfo.syzd = edittime
+                                ktinfo.save()
+                            msg = monster_info.monstername + "的击杀时间已修改为：\n" + edittimestr
+                        except ServersMonsterKillTime.DoesNotExist:
+                            msg = "[error]"
+                    except MonsterList.DoesNotExist:
+                        msg = "目前只支持S级狩猎怪"
+                except IndexError:
+                    msg = "狩猎时钟edit命令示例：\n/hunt edit [怪物名称] [时间]\n时间格式例：\n2019-07-22 02:44:45"
+
+        except HunterGroupID.DoesNotExist:
+            msg = "该群并非狩猎组群组"
+        reply_action = reply_message_action(receive, msg)
+        action_list.append(reply_action)
+        return action_list
+    except Exception as e:
+        msg = "Error: {}".format(type(e))
+        action_list.append(reply_message_action(receive, msg))
+        logging.error(e)
+        traceback.print_exc(e)
+    return action_list
+
```

Comment:
This file is much more than complicated, which should be refactored.
I already need to refactor lots of garbage code of former commands lol, so new commands should be written well to prevent us from a catastrophy in later maintaining.